### DISCUSSION TITLE
Proposal to change ADempiere (English version)

### DISCUSSION THREAD

**Discussion by eduardolopezuy** [adempiere/adempiere#4131] [ADempiere's New Launching]
I propose the headlines of some ideas that in my opinion will serve to promote the ADempiere project. Some of them have already been written by Alberto and Ramiro.

**ADempiere project**

- What unites us to the few members of the project that we are currently in, is the dream of building the best ERP software of business quality (SMB).
- The ADempiere project is not just a technical project, it is functional and documentation.
- Luckily many citizens previously laid the foundations and we must continue with it, making this project more reliable, more modern, more open, more collaborative, more distributed and better financed.
- We must make it very explicit what the common dream is and all actions must be aligned to this dream.

**Governance**

- Simplify the government of the project by eliminating the figure of the association (foundation) and empower the board and the general assembly of citizens. https://www.adempiere.io/foundation/statutes.html
- Simplify the statutes.
- Propose schedule with future steps.
- Publish on the web participants of its bodies, minutes of meetings, decisions and balances.

**Roadmap**

- Discuss and publish the technical, functional and documentary roadmap

**Financing of the ADempiere Project**

- This is vital to ensure the future of the project.
- It should not fall only on its citizens
- We must ensure that companies finance the ADempiere project in the majority
- But today, the responsibility for financing lies with the citizens, since the project is disorganized and there is only us. Therefore, it is not possible to reduce the amounts.

**Other contributors**

- To attract new contributors we must provide small monthly payments (Patreon) without the right to vote.

**Project finance**

- They must be aligned to the ADempiere roadmap
- Anyone can present projects to the community for the purpose of seeking co-financing or full financing.
- The decision of the convenience of co-financing with money from the community and the percentage will be decided by the board by majority.
- Depending on the size of the project, these will be the requirements that the bidder must meet. The sizes will be small, medium and large. The requirements for each type of project must be stipulated in writing.

**Comment by juddm** [comment]
Hi Eduardo,

I think you raise some good points about how to move ADempiere forward.  

In regards to finances, I think we should open the conversation to how we can finance the project.  My hypothesis is that a small number of the users of the project will provide revenue streams to the project team so to be sustainable, we need a large user base.  We can think about how value is created and realised in the community.

1. User Organizations use ADempiere to run their affairs
2. Some User Organizations want support to learn, implement, customise and/or support their deployment
3. Citizens develop, customise, extend, document, support, integrate, demo, evaluate, promote/evangelize & host ADempiere for User Organizations

Many activities do not have value streams if we keep an open & free for all project. e.g. documentation  Therefore, we need to provide services that customers want to pay for and use these revenue streams to subsidise citizens/community providing resource to the project.  Secondly, we need the User base to be large enough to generate enough demand for services.

Here are some services we could offer that would keep the community free & open, but also provide revenue streams to fund citizens to develop, customise, extend, document, support, integrate, demo, evaluate, promote/evangelize & host
1. User Organization support (currently this is provided directly by citizens & their own businesses)
2. Online Training: Administrator, End User & Specialist Topics (I understand there have been some businesses that have done this)
3. Books: There have been a few guides
4. Advertising: perhaps embedded in video blogs, web pages
5. Customization / Bounties: would be offered by User Organizations ?
6. Pre-Implementation Assessments
7. Implementation Assurance
8. Implementation Methodology / Online Tools 
9. Data Migration / Transformation

This is not supposed to be a complete list, but just an illustration of a way to analyse the potential ways to support the community.  Ultimately, we need to encourage more adoption and I would focus on lowering the bar for user organizations to evaluate and adopt ADempiere.  

As an example, we could do this by creating an official hosted service and charge a small markup on top of the hosting platform - which would be contributed back to the community/foundation.  People would have the option of deploying themselves or using the hosted service. Over time, we could integrate different services with the hosted platform (e.g. document scanning & indexing, planning) to enrich the offering

Just sharing some thoughts and I would be happy to structure an online session where we could analyse, evaluate, prioritise and agree an action/ownership plan - but we probably need a little more debate and ideas before we did this.

Regards,

Mike

**Comment by yamelsenih** [comment]
Hello, this is a great start and we can move ADempiere but is very important define all details about it.

**Comment by eduardolopezuy** [comment]
Hi Mike, thanks for your thoughts. They are always very valuable.

I agree with what you write.
To finance the OS ADempiere project we need to provide services that customers value and are willing to pay for. With this flow of resources, the realization of your roadmap and promotion is encouraged, creating a virtuous circle.

The ADempiere code will always be free and open, but as you say, there are many ancillary content or services surrounding the ADempiere code that can be delivered for a price.
 
But what is not clear to me and I would like to know your opinion, is what should be the formal organization of the entity that should be behind the commercial activities of the OS ADempiere project.
And that is where many questions arise.

What type of company should be the one that delivers these services? Is it the same non-profit ADempiere Association that should do it?
What formal relationship should this organization have with the ADempiere project?
What body should run it?
How is the creation of receivable resources initially financed?
How are content creators paid? Sharing with the creator its price?
In addition, to perform hosting, support or Implementation Assurance, a team of people with responsibilities that are paid is required.
How to solve the current competition problem with what citizens' organizations do today?

All these ideas are aligned to recognize that for a project to grow vigorously it must be properly financed not only by its membership, but also by delivering valuable services to those who want to pay for it. And for this we cannot depend only on the resources contributed by the community 100% honoraria.
Regards
Eduardo

**Comment by juddm** [comment]
Hi Eduardo,

Thanks for your thoughtful comments.

My view would be we could agree some services that make sense to do
centrally (things that should be done once and that can scale).

I would expect the Foundation does this so the revenues flow to the not for
profit - and then the foundation can pay the content creators.

I think this allows space for citizens and their businesses to also fund
their own activities (e.g.support, implementation, customer specific
customisation etc)

We could also think about services that those support organisations might
also need or want (e.g. assurance, project scoping & tracking)

I do see the chicken and egg problem of getting started.  We probably need
to create a plan of what we do when, and link this to leading indicators so
that we can be confident we are growing the community and creating the
market for the services we choose.

Ill try to put together a more structured value stream model which should
help us identify some services that would have value to customers and
support businesses, scale, be low resource intensive and have low capital
requirements.

I'm sure I'll discover this just behind a unicorn.... But at least we can
use the model to ensure we've evaluated all the options and then make the
best choice.

We may learn something through the process, or we may decided to do nothing
but it is at least a structured way to think through the problem.

Give me a week and I will post something...

Mike

**Comment by e-Evolution** [comment]
Hi Community!

The financing of the foundation is a complex issue since it is specified in the statutes https://www.adempiere.io/foundation/statutes.html that it is a non-profit organization, but on the other hand it is the custody of the project's assets.

In our past civil war, a trigger for the conflict was not having a policy for the use of the brand, something that came later, beware, this is not set in stone, it could change and be voted on by the citizens.

https://wiki.adempiere.net/Brand_Usage_Rules

So the financing alternatives are limited.

The fact that the foundation is a non-profit organization allowed it not to compete directly with the commercial interests of sponsoring members themselves who offer their contribution to the foundation, on the other hand, since the foundation is a non-profit organization, it allows transparency and neutrality towards the community and contributors.

I think that the objectives of the foundation established in its statutes are fine and laudable, but this is where we come to the same question, how do we finance the foundation? without conflicting with sponsoring members and maintaining neutrality and transparency.

I know that in other foundations in charge of open source projects they are financed by their members and they achieve this because there are so many of them. It would be interesting to investigate how they are financed, for example.https://dotnetfoundation.org/  , I can assume that the main funding of this foundation comes from Microsoft but not exclusively.

Here are some ideas that we could put into practice:

1.- Given that the foundation is the owner of the brand and intellectual property, it is possible that the foundation grants licenses for use and through these licenses rights are granted to sponsors for their legal and correct use established by the foundation, this would allow anyone who pays for the license or certifications that they can use for the use of the brand under the guidelines of the foundation, this already works in the music industry allowing the collection of royalties that allow the foundation to be financed and thus reward the creators (Certifications, Manuals, Training material and why not developments)

2.- That the foundation can obtain funds for the issuance of certificates granted by accreditation or exams (Certifications that accredit knowledge of the use of the solution, both at a technical and functional level, thus guaranteeing standardization and high quality of knowledge of the platform) , the income from this concept should allow payment for the creators, maintainers of the teaching material and certification with a model similar to how YouTube works today, which pays its platform creators, this model also works in Apple and its ecosystem where developers they have a 70/30 share 70 for the developer 30 for Apple. These are some ideas. If someone knows other mechanisms that may work, it would be good to hear them.

3.- Perhaps have an association that is friendly to the foundation that pays the rights to use the license to use the brand to the foundation through a partner affiliation and their respective certifications granted by the accreditation of courses or exams as happens in universities private and in this way it can function as an entity that offers high-value services aligned with the commercial interests of all those who participate in the ADempiere ecosystem but without being in conflict with the objectives of the foundation.

I remember that in the past an attempt was made to create ABC (ADempiere Business Consultants) as an association of consulting companies worldwide whose objectives were to promote ADempiere with a commercial focus, offering high-value services to the global market, allowing ADempiere to compete against proprietary brands. With a global presence, the main idea is a large marketplace with well-defined rules allowing open competition and offering quality services, alienating common interests without each company losing its autonomy and independence, unfortunately it did not come to fruition, not because it did not work if not because it was never put into practice.

The key is that the association of commercial companies does not have conflicts with the goals of the foundation, but instead is the main ally and financing mechanism through the payment of brand use licenses to partners, perhaps the community today. she's mature enough to try this again.

On the other hand, the conflicts of the past left the ADempiere brand very hurt, I want to think that this was the reason why many sponsors decided to make their own fork and build their own brand since marketing ADempiere with all the noise around the conflict was complicated.

see story here https://github.com/adempiere/adempiere/discussions/4133

For this reason we should evaluate the possibility of rebranding and jointly try to position a new brand where the foundation has complete control of the rights and domains.

I still remember frightened clients when they entered the hijacked domain adempiere.com or adempiere.org causing noise and mistrust with a clear intention to cause deceit by the perpetrators.

So thinking about a rebranding would not be a bad idea, perhaps opening the discussion of this and listening to those who have commercial experience can be interesting.

Finding the balance between the interests and incentives of the contributors represented by the ADempiere foundation, the commercial participants and the market represented by the clients allows to create an ecosystem where everyone can obtain a win-win relationship.

I am open and motivated to work on something that I can achieve.

What do you think? Is it possible to align the interests and incentives of all those involved?

kind regards
Victor Perez 
ADempiere founder & Foundation board directory member

**Comment by juddm** [comment]
Hi All,

Firstly, I think it is important we find common ground on what we can move
forward with together.  Otherwise we are stuck where we are.  My intentions
are to make the project stronger and ensure there are resources to make it
sustainable. The last thing we need is a civil war !

The foundation is a not for profit - but that does not mean it cannot
provide 'commercial' services or compete against commercial businesses.  I
am not suggesting we have the foundation compete against businesses but to
find some services it can provide at scale, low cost and where there is a
natural advantage as the owner of the IP.

If we can evaluate the options, and find some services that would make
sense - then this would at least provide some funding for foundation
activities and benefit and build the community.

Regards

Mike

**Comment by e-Evolution** [comment]
Reviewing the history of the project https://github.com/adempiere/adempiere/discussions/4133 , I remembered the ABC initiative and what could have been if it had not been dissolved in favor of avoiding the civil war, even with this it was not possible to avoid it, 

https://sourceforge.net/p/adempiere/discussion/611167/thread/bf89a4e3/
https://www.prlog.org/10693986-adempiere-implementers-form-an-alliance-for-worldwide-service-offering.html
https://www.openpr.com/news/132620/adempiere-implementers-form-an-alliance-for-worldwide-service-offering.html
https://www.heise.de/news/Support-fuer-ERP-Loesung-Adempiere-1008018.html

On the other hand, it exposed the true intentions of controlling the project by a single person. 
 
I am convinced that after the smoke and noise dissipated we should resume the initiative, it is no coincidence that those of us who are in the ADempiere project today are the majority who started the initiative.

@juddm Mike, @rvergara Ramiro, @marcalwestf Mario what do you think? I am convinced that we could revive ADempiere Business Consultants (ABC), I guess Mike has all the required information on the company founded in England and we can activate it and invite consulting firms to join.

I think we can achieve harmony between the foundation that custodians the assets and the association that is focused on the commercial interests that ABC originally applied for through a trademark license authorized by the foundation and its citizens.

Mike designed this schematic that we can improve 

![image](https://github.com/adempiere/adempiere/assets/786968/c33eff75-880c-40ca-b06b-82d7516640a9)

What do you think?

kind regards
Victor
